src/sentry/static/sentry/app/components/events/interfaces/spans/spanTree.tsx

```diff
@@ -335,4 +340,18 @@ const TraceViewContainer = styled('div')`
   border-bottom-right-radius: 3px;
 `;
 
+function isJavaScriptSDK(SDKName: string | undefined): boolean {
+  if (!SDKName) {
```

Comment:
This casing makes me think it is a constructor/class. What about `sdkName`?

Reply:
https://github.com/getsentry/sentry/pull/17887/commits/57f9dae1658689d7bc98108ba1b4a51c13ce8f3e